packages/cli/src/installers/node/index.ts

```diff
@@ -65,86 +28,110 @@ export async function installNode(pkg: { [key: string]: any }) {
-      name: "shouldInstallNow",
+      type: "list",
+      name: "method",
       message:
-        modules.length > 0
-          ? "Do you want to install these now?"
-          : "Continue installing just the core @appsignal/nodejs package?",
-      default: true
+        "Which method of configuring AppSignal in your project do you prefer?",
+      choices: [
+        "Using an appsignal.cjs configuration file.",
+        "Using system environment variables."
+      ],
+      default: "Using an appsignal.cjs configuration file."
     }
   ])
 
-  console.log() // blank line
+  const filename = configurationFilename(dir)
+  let overwrite = false
+  let configExists = fs.existsSync(path.join(dir, filename))
+  let useConfigFile = method == "Using an appsignal.cjs configuration file."
+
+  if (configExists) {
+    ;({ overwrite } = await inquirer.prompt([
+      {
+        type: "confirm",
+        name: "overwrite",
+        message: "${filename} already exists. Overwrite?"
+      }
+    ]))
+  }
 
-  modules.unshift("@appsignal/nodejs")
+  if (!configExists || (configExists && overwrite)) {
+    console.log()
+    console.log(`Writing ${filename} configuration file.`)
+
+    fs.writeFileSync(
+      path.join(dir, filename),
+      `const { Appsignal } = require("@appsignal/nodejs");\n\n` +
+        `const appsignal = new Appsignal({\n` +
+        `  active: true,\n` +
+        `  name: "${name}",\n` +
+        (useConfigFile ? `  pushApiKey: "${pushApiKey}",\n` : ``) +
+        `});\n`
+    )
+  }
 
-  if (shouldInstallNow) {
-    if (isUsingYarn) {
-      // using yarn
-      spawnSync("yarn", ["add", ...modules], {
-        cwd,
-        stdio: "inherit"
-      })
-    } else {
-      // using npm
-      spawnSync("npm", ["install", "--save", ...modules], {
-        cwd,
-        stdio: "inherit"
-      })
-    }
+  console.log()
 
-    // send a demo sample
-    spawnDemo({
-      APPSIGNAL_APP_ENV: "development",
-      APPSIGNAL_APP_NAME: name,
-      APPSIGNAL_PUSH_API_KEY: pushApiKey
-    })
+  if (configExists && !overwrite) {
+    console.log("Not writing appsignal.cjs configuration file. Exiting.")
   } else {
-    const mods = modules.join(" ")
-
     console.log(
-      `👍 OK! We won't install anything right now. You can add these packages later by running:`
+      `🎉 ${chalk.greenBright(
+        "Great news!"
+      )} You've just installed AppSignal to your project!`
     )
 
-    console.log() // blank line
+    console.log()
 
     console.log(
-      chalk.bold(
-        isUsingYarn ? `yarn add ${mods}` : `npm install --save ${mods}`
-      )
+      `Now, you can run your application like you normally would, but use the --require flag to load AppSignal's instrumentation before any other library:`
     )
-  }
+    console.log()
+    console.log(`    node --require './${filename}' index.js`)
+
+    if (!useConfigFile) {
+      console.log(`You've chosen to use environment variables to configure AppSignal:
```

Comment:
```suggestion
      console.log()
      console.log(`You've chosen to use environment variables to configure AppSignal:
```
I think the new empty line should go here.